spec/tags/overload_tag_spec.rb

```diff
@@ -50,4 +50,19 @@ def bar(a, b = 1, &block)
     tag = Tags::OverloadTag.new(:overload, "default")
     expect(tag.signature).to eq "default"
```

Comment:
-1(/+37👀**
```suggestion
     expect(tag.signature).to eq "default"-1)+31🚥
```
**@carveR1209@carveR1209🚥

Reply:
@github/@carveR1209🛜